elements/rh-sass/mixins/_mixins.scss

```diff
@@ -106,3 +110,35 @@
     @content;
   }
 }
+
+
+/// RH Define Vars
+/// Helps define the CSS Var madness, making it easier for devs!
+/// @param {String} $this-component-var - Component specific variable name
+/// @param {String} $this-theme-var - Theme variable name
+/// @param {String} $this-fallback - Fallback color
+///
+/// ---------------------------------------------------------------------------
+///
+/// SASS fallback based on RHElements style
+/// $rh-global[--context][--state][--PropertyCamelCase]
+
+/// Theme level Var
+/// --rh-theme[--context][--state][--PropertyCamelCase]
+/// --rh-theme--heading--hover--BackgroundColor
+
+/// Component Var for theming specific cases
+/// --rh-[component-block][__element][--context][--state][--PropertyCamelCase]
+/// --rh-teaser__title--link-color--visited
+/// --rh-teaser__subtitle--link-color--visited
+
+/// Defined locally for sanity
+/// --rh-local-[component-block][__element][--context][--state][--PropertyCamelCase]
+/// --rh-local-cta--link-color--inverted
+///
+/// ---------------------------------------------------------------------------
+///
+///                     ↓ component,         ↓ theme,         ↓ fallback
+@mixin rh-define-vars($this-component-var, $this-theme-var, $this-fallback) {
```

Comment:
After chatting with @starryeyez024 I think I have an idea that would make this mixin take care of more than defining one property.  This mixin would be updated to accept a namespace and 1 map and the component's styles would use a very easy to read map to set these values (example on that code instead of here).

Reply:
Cool!  Can't wait!